src/Common/src/Interop/Windows/mincore/Interop.Manual.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Rather than introducing a new file, the project should just use src\Common\src\Interop\Windows\mincore\Interop.Errors.cs and augment it with any needed values that may be missing.